appside/frames/body.js

```diff
@@ -215,15 +218,39 @@ class BodyMapFrame extends Frame {
             $(label).text(stmt);
             right.appendChild(label);
             right.appendChild(document.createElement('br'));
+
+            this.user_input.set(name, 'false'); // all unchecked
         }
 
+        // next will be implemented in navigator?
+        let next = document.createElement('button');
+        $(next).attr('class', 'bodymap_button');
+        $(next).text('Next');
+        $(next).click(function() {
+            var choices = document.getElementsByTagName('input');
+            for (let each of choices) {
+                if (each.checked) {
+                    this.user_input.set(each.id, 'true');
```

Comment:
this is not quite what we were looking for.

According to #30, `user_input` should contain entries like
` "its purr": true`
where the key is literally the text of the question

This code creates entries like 
`"label_0": true`
which will not be understood by the model. (This would become evident upon integration testing.)

We need the text of the question, not the checkbox ID.

Reply:
Changed each.id to each.dataset.text by adding a custom dataset attribute when initializing checkbox elements. This allows access of text of answer choice through each.dataset.text.

Reply:
looks good